docs/visualization/path/response_time.md

```diff
@@ -43,3 +51,31 @@ plot.show()
 ```
 
 ![response_time_worst_histogram](../../imgs/response_time_worst_histogram.png)
+
+## Stacked Bar
+
+The following scripts generate stacked bar graphs of response time or path latency.
```

Comment:
The word "path latency" suddenly appears and it may confuse a reader.
Are the followings the reason why you wanted to add "or path latency"?
- A path is not always end-to-end path. For instance, a path can be from the beginning of planning component to the end of planning component
- In this case, the time should not be called as "response time" , but just a "path latency"

If that's the case, we can say the same thing for histogram, can't we?
My suggestion is adding the following excuse just after "In context of CARET, response time..." sentence at the beginning of this page. Or adding admonition somewhere in the page.

---
"Response time" is for a system with input/output. Therefore, when analyzing a part of a system (when a target path is a part of end-to-end path), it should be referred to as "path latency" rather than "response time". However, for simplicity both will be referred to as "response time" in this document.

「Response timeは厳密には入出力を持つシステムに適用されるものである。そのため、システム内の一部のパスを解析する場合にはresponse timeではなくpath latencyと呼ぶべきである。しかし、本ドキュメントでは簡単のためいずれもresponse timeと呼ぶ」

Reply:
@takeshi-iwanari 
Thank you.
I think you are right and response time is the time diff from input to output of the target **system**.
I added your suggestion in response_time.md.